src/views/viewer/HomeView.tsx

```diff
@@ -64,4 +65,8 @@ const Container = styled.div`
 const StyledInfiniteVideoGrid = styled(InfiniteVideoGrid)`
   margin: 0;
   padding-bottom: 4rem;
+
+  :not(:first-child) {
```

Comment:
Im getting 
![image](https://user-images.githubusercontent.com/51168865/126770193-46fa094a-087f-49fc-9493-a9ce1fe2e6a9.png)
I mean, is it possible to change this to `:first-of-type`?